### ISSUE TITLE
canActivateAR incorrectly returning false on iOS 16 chrome/firefox

### ISSUE THREAD

**Issue by CameronCrane** [google/model-viewer#3860]
### Description

It seems that with the release of iOS 16, the canActivateAr method incorrectly returns false on iOS 16/chrome and iOS 16/Firefox, even if manually setting `arModes="webxr scene-viewer quick-look"`. On the same device (using an iPhone 13 Pro in this case), it returns true _only_ when using Safari. 

#### Live Demo
Here are some snippets of our implementation.

```
public get canActivateAR() {
      return this[$modelViewer].value
        ? this[$modelViewer].value.canActivateAR
        : false;
  }
```

```
 private [$onArButtonClick]() {
      if (this.canActivateAR) {
        alert('can activate AR')
        return this[$modelViewer].value?.activateAR();
      }
      alert('cannot activate AR');
  }
```

### Version
- model-viewer: v2.0.0 (currently most recent)

### Browser Affected
- [X] Chrome
- [ ] Edge
- [X] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [X] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR
- [X] WebXR
- [X] SceneViewer?
- [ ] QuickLook

**Comment by jediskej** [comment]
I'm having the same problem on Android. Any solutions?

**Comment by elalish** [comment]
Indeed, this is an iOS regression and was worked-around in v2.0: https://github.com/google/model-viewer/pull/3784
